src/runner.py

```diff
@@ -33,109 +33,19 @@ class Runner:
     # mode_jsons = "running_with_jsons"
     # mode_run_config = "running_with_config"
 
-    def __init__(self, directory_with_jsons: Optional[Path] = None):
-        #     self.run_config = run_config
-        #     self.training_instances = None
-        #     self.test_instances = []
+    def __init__(self, directory_with_jsons: Optional[Path] = None, data_directory_path: Optional[Path] = None):
         self.directory_with_jsons = directory_with_jsons
+        # TODO make this a parameter
+        all_instances = [
+            os.path.join(data_directory_path, x) for x in os.listdir(data_directory_path)
+        ]
+        # Shuffle the instances
+        random.shuffle(all_instances)
+        # Do random split here
+        self.training_instances = all_instances[: int(len(all_instances) * 0.8)]
+        self.test_instances = all_instances[int(len(all_instances) * 0.8) :]
+        self.val_instances = None
```

Comment:
i am not saying it should live in the runner but the way it was before we had different `TestCases` running with different train/test dataset. This way the split is fixed